CMakeLists.txt

```diff
@@ -986,7 +986,14 @@ set(MIXXX_INSTALL_BINDIR ".")
 set(MIXXX_INSTALL_DATADIR ".")
 set(MIXXX_INSTALL_DOCDIR "./doc")
 set(MIXXX_INSTALL_LICENSEDIR "./doc")
-if (UNIX)
+if (APPLE)
+  set(MIXXX_INSTALL_BINDIR "${CMAKE_INSTALL_BINDIR}")
+  set(MACOS_BUNDLE_NAME Mixxx)
+  set(MIXXX_INSTALL_PREFIX "${MACOS_BUNDLE_NAME}.app/Contents/Resources")
```

Comment:
I would expect `MIXXX_INSTALL_PREFIX=${MACOS_BUNDLE_NAME}.app` without the subdirectories? Then `MIXXX_INSTALL_PREFIX` could be reused in `fixup_bundle` (see below) where this path is currently duplicated.

Reply:
good idea